Makefile

```diff
@@ -104,7 +104,7 @@ test:
 ############################################################
 
 coverage:
-	@common/scripts/codecov.sh
+	@common/scripts/codecov.sh ${BUILD_LOCALLY}
```

Comment:
Do you want to add a default value for `BUILD_LOCALLY`?

Reply:
Sorry Just noticed that I forgot to set it.